google/cloud/forseti/scanner/scanners/config_validator_scanner.py

```diff
@@ -0,0 +1,190 @@
+# Copyright 2019 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Config Validator Scanner."""
+
+from google.protobuf import json_format
+
+from google.cloud.forseti.common.util import logger
+from google.cloud.forseti.scanner.scanners import base_scanner
+from google.cloud.forseti.scanner.scanners.config_validator_util import cv_data_converter
+from google.cloud.forseti.scanner.scanners.config_validator_util import validator_client
+from google.cloud.forseti.services.model.importer import importer
+
+
+LOGGER = logger.get_logger(__name__)
+
+
+class ConfigValidatorScanner(base_scanner.BaseScanner):
+    """Config Validator Scanner."""
+
+    violation_type = 'CONFIG_VALIDATOR_VIOLATION'
```

Comment:
I agreed with your point, this will need to make changes to the notifier in order for it to notify properly, will require more design to implement this piece